media/CircleGraph/index.js

```diff
@@ -167,6 +170,16 @@ host.BrowserHost = class {
         case "visq":
           this._msgVisq(message);
           break;
+        case "changemode":
```

Comment:
why is this needed?

Reply:
After .visq file  was loaded (to enable visq_style), we need to reload html, the only options i found, was 
1) to dispose current panel
2) to open it with another mode, (so that __viewMode is changed only while creating html),
but it seemed to be not that fast in comparison with with direct notify message about changed _mode.

Reply:
I don't understand the current scenario. I think @jinevening 's question should be answered first.

Reply:
The tool is intended to provide handy creation of .json suitable for mixed precision quantization.
1) we need to choose `circle` model for creation of empty json with empty layers
2) then we choose NN layers to set quantization parameters different from their default values
3) to make selection easy graphical interface ('netron') was provided 
4) in case we have .visq file we may use it to make selection of 'erroneous' layers more convenient.

Reply:
I still don't understand why changemode is needed.

Reply:
> I still don't understand why changemode is needed.

Sorry. To make it clear:
after `visq` data is loaded i use `setMode("visqselr")` but `host.BrowserHost` from `index.js` (which is used in `view.js` to distinguish which color-styles should be attached) still has its `this._mode` set to `selector` so no `visq` color-styles were shown.

Reply:
Sorry but I don't understand just by explaining by words.
If possible, please describe in some readme.md file that users can follow, in this draft.
These are my questions I got.
- what is the purpose of this?
- what does the user get from this?
- what does the user have to do with the result?
- when does the mode change to visqselr?
- what do this program expect from visqselr?
- how does the user get back to this?

Reply:
ok.